openmdao/utils/om.py

```diff
@@ -442,6 +444,47 @@ def _cite(prob):
     _load_and_exec(options.file[0], user_args)
 
 
+def _show_dependency_versions():
+    import openmdao
+
+    setup_path = os.path.realpath(os.path.dirname(os.path.dirname(openmdao.__file__))) + "/"
```

Comment:
minor detail but you don't need to add the "/" here because you are using `path.join` in the next line